common/src/main/java/com/wynntils/models/experience/CombatXpModel.java

```diff
@@ -48,7 +50,7 @@ public CombatXpModel(WorldStateModel worldStateModel) {
     }
 
     @SubscribeEvent
-    public void onExperienceGain(CombatXpGainEvent event) {
+    public void onXpGain(CombatXpGainEvent event) {
```

Comment:
Unless you changed all of these names in this PR, I don't think this is generalized

Reply:
I meant that I don't think all methods use Xp instead of Experience. But we don't need to fix it now.

Reply:
I checked, and at least in CombatXpModel all use XP instead of experience.